vignettes/ggPMX-ggplot.Rmd

```diff
@@ -0,0 +1,81 @@
+---
+title: "How to use ggplot2 syntax with ggPMX"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to use ggplot syntax with ggPMX}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+We will create a simple ggPMX controller using the theophylline example:
+
+```{r, warning=FALSE, message=FALSE}
```

Comment:
This is achieved in the other parts of the `ggPMX` packge by `knitr::opts_chunk$set(out.width = "100%", warning = FALSE, message = FALSE)`  the advantage to this approach is you can see all warnings without having to modify each source block in the code.

Reply:
Thank you, will change.